pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -92,7 +93,14 @@ func ConvReason(status libvirt.DomainState, reason int) api.StateChangeReason {
 }
 
 func SetDomainSpecStr(virConn cli.Connection, vmi *v1.VirtualMachineInstance, wantedSpec string) (cli.VirDomain, error) {
-	log.Log.Object(vmi).V(3).With("xml", wantedSpec).Info("Domain XML generated.")
+
+	domxml, err := convertToLibvirtXML(wantedSpec)
+	if err == nil {
+		fmt.Println("Domain XML generated.")
+		fmt.Println(domxml)
```

Comment:
Maybe we cannot use structured logs from k8s but we can still use golang log package to get at least the timestamps

https://golang.org/pkg/log/

Also add the vmi name to it so it's easier to track.

Reply:
@qinqon k8s logger uses golang log and overrides the IO writer. So logging will still occur in JSON format with escapes. Looks like Logrus logging package can be used instead.

Reply:
@qinqon @rmohr Why can't I just print it with fmt.Println?

Reply:
Perhaps all we need is something like [1]?
It's very small and seems to do the job. Then we could simply log the output.
What do you think?

[1] https://github.com/go-xmlfmt/xmlfmt/blob/master/xmlfmt.go

Reply:
@vladikr I've suggested that package in the initial PR phase. Later on together with @danielBelenky we decided to use a more maintained package libvirt-go-xml.